src/Compilers/CSharp/Portable/Compiler/MethodBodySynthesizer.Lowered.cs

```diff
@@ -149,7 +149,7 @@ internal override void GenerateMethodBody(TypeCompilationState compilationState,
                 LabelSymbol start = F.GenerateLabel("start");
 
 
-                //  This method should be kept consistent with SynthesizedStringSwitchHashMethod.ConstructStringHash
+                //  This method should be kept consistent with SynthesizedStringSwitchHashMethod.ComputeStringHash
```

Comment:
Sorry if I missed this the first time. It seems like it would be best to use a doc comment and cref for this, if the method being referenced is accessible from here.